internal/services/containers/kubernetes_cluster_data_source_test.go

```diff
@@ -577,6 +578,23 @@ func testAccDataSourceKubernetesCluster_addOnProfileOpenServiceMesh(t *testing.T
 	})
 }
 
+func testAccDataSourceKubernetesCluster_addOnProfileAzureKeyvaultSecretsProvider(t *testing.T) {
```

Comment:
This should also be public
```suggestion
func TestAccDataSourceKubernetesCluster_addOnProfileAzureKeyvaultSecretsProvider(t *testing.T) {
```